merlin/schema/io/proto_utils.py

```diff
@@ -22,11 +22,43 @@
 ProtoMessageType = TypeVar("ProtoMessageType", bound=BetterProtoMessage)
 
 
-def has_field(self, field_name):
-    return betterproto.serialized_on_wire(getattr(self, field_name))
+def has_field(message: ProtoMessageType, field_name: str) -> bool:
+    """Check if a Protobuf message has a particular field
+
+    Parameters
+    ----------
+    message : ProtoMessageType
+        Protobuf message object
+    field_name : str
+        Name of the field to look for
+    message: ProtoMessageType :
+
+    field_name: str :
+
+
+    Returns
+    -------
+    bool
+        Whether or not the named field exists on the message object
```

Comment:
This might sound pithy, but can you be even more explicit and include `true` or `false`?

> `True` if the named field exists on the message object